src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar main-search-bar"
+      data-qa="big"
+    >
+      <input
+        class="search-bar-input search-bar-input--main"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
```

Comment:
The `form` tag is typically used to collect user input. If your `form` tag doesn't contain any `submit` button, it might be better to use a `div` or `section` tag instead. This is not a code quality issue, but a semantic one. Semantic HTML is important for accessibility and SEO.